### PR TITLE
Further additions to pyscript documenntation for Issue #644

### PR DESCRIPTION
**Pull request by anselor**

### CODE REVIEW

File: docs/features/scripting.rst [hunk 1]
```diff
@@ -94,6 +94,24 @@ where:
 * ``command`` and ``args`` are entered exactly like they would be entered by
   a user of your application.
 
+Advanced Support
+~~~~~~~~~~~~~~~~
+
+When implementing a command, setting ``self.__last_result`` allows for application-specific
```
Comment (tleonhardt) [inline docs/features/scripting.rst:100]: I think this is incorrect and should be `self.last_result` (applies below as well)

File: docs/features/scripting.rst [hunk 2]
```diff
@@ -94,6 +94,24 @@ where:
 * ``command`` and ``args`` are entered exactly like they would be entered by
   a user of your application.
 
+Advanced Support
```
Comment (tleonhardt) [inline docs/features/scripting.rst:97]: This section should go at the end of the file.  As is, there is an extra line that is completely out of place.

Comment (anselor) [reply docs/features/scripting.rst:97]: I don't know what you mean by this. This is already the last section in this file.

Comment (tleonhardt) [reply docs/features/scripting.rst:100]: I pushed a small change to address what I was talking about

**Pull request merged**